gitget.gemspec

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
+require 'gitget/version'
+
+Gem::Specification.new do |s|
+  s.name        =  'gitget'
+  s.version     =  Github::VERSION
+
+  s.summary     =  'Gets public information about Github developers'
+  s.description =  'Extracts developer statistics'
+  s.authors     =  ['Renaud Jollet', 'Nicholas Danks', 'Sami Larrousi Tribek']
+  s.email       =  ['rjollet@gmail.com', 'nicholasdanks@hotmail.com']
```

Comment:
my email is renaudjollet@neurow.org @samilaaroussi what email do you want to use ?